### DISCUSSION TITLE
Kann man einen Reboot via HTTP-Request durchführen?

### DISCUSSION THREAD

**Discussion by gsbox** [frickelzeugs/FingerprintDoorbell#99] [General]
Manchmal dauert das senden des MQTT Requests lange. Ein Neustart behebt das Problem. Das würde ich gerne Automatisieren

**Comment by Noschvie** [comment]
Ursache ähnlich wie in discussion [62](https://github.com/frickelzeugs/FingerprintDoorbell/discussions/62) ?
